src/Snowflake/Connection.php

```diff
@@ -46,6 +46,7 @@ public function __construct(array $options)
 
         $port = isset($options['port']) ? (int)$options['port'] : 443;
         $tracing = isset($options['tracing']) ? (int)$options['tracing'] : 0;
+        $maxBackoffAttempts = isset($options['maxBackoffAttempts']) ? (int)$options['maxBackoffAttempts'] : 5;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/569139/20266362/1c741a1a-aa76-11e6-9c52-87b3154a8f68.png)

yes, it wasn't checked by default